src/js/hca/containers/IntroductionPage.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+import { focusElement } from '../../common/utils/helpers';
+import OMBInfo from '../../common/components/OMBInfo';
+import FormTitle from '../../common/schemaform/FormTitle';
+import SaveInProgressIntro, { introActions, introSelector } from '../../common/schemaform/SaveInProgressIntro';
+
+class IntroductionPage extends React.Component {
+  componentDidMount() {
+    focusElement('.va-nav-breadcrumbs-list');
+  }
+
+  render() {
+    return (
+      <div className="schemaform-intro">
+        <FormTitle title="Apply for health care benefits"/>
+        <p>Equal to VA Form 10-10EZ (Application for Health Benefits).</p>
+        <SaveInProgressIntro
+          messages={this.props.route.formConfig.savedFormMessages}
+          pageList={this.props.route.pageList}
+          resumeOnly
+          {...this.props.saveInProgressActions}
+          {...this.props.saveInProgress}/>
+        <div className="process schemaform-process">
+          <ol>
+            <li className="process-step list-one">
+              <div><h5>Prepare</h5></div>
+              <div><h6>To fill out this application, you’ll need your:</h6></div>
+              <ul>
+                <li>Social Security number (required)</li>
+                <li>Copy of your military discharge papers (DD214 or other separation documents)</li>
+                <li>Financial information—and your dependents’ financial information</li>
+                <li>Most recent tax return</li>
+                <li>Account numbers for any health insurance you currently have (such as Medicare, private insurance, or insurance from an employer)</li>
+              </ul>
+              <p><strong>What if I need help filling out my application?</strong> An accredited representative with a Veterans Service Organization (VSO) can help you fill out your claim. <a href="/disability-benefits/apply/help/index.html">Find an accredited representative</a>.</p>
+              <div><h6>Vision and dental benefits</h6></div>
```

Comment:
It looks like this should be in a blue box.

Reply:
We decided the remove the blue box for now.